lib/seafoam/cfg/disassembler.rb

```diff
@@ -1,5 +1,6 @@
 require 'stringio'
 require 'zlib'
+require 'crabstone'
```

Comment:
You need to declare a dependency on `crabstone` in the `gemspec` file.

Reply:
:( my attempt to declare unsuccessful